src/main/java/com/ibm/cloud/sdk/core/util/TypeAdapterRuntimeTypeWrapper.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (C) 2011 Google Inc.
```

Comment:
Yep, will add that

Reply:
I'll also add the Google copyright prolog to the DynamicModelTypeAdapterFactory class since some of that code was modeled after a couple of the Gson internal type adapters.